test/source/mock/google/strategies/send-message-strategy.ts

```diff
@@ -472,6 +472,8 @@ export class TestBySubjectStrategyContext {
       this.strategy = new SaveMessageInStorageStrategy();
     } else if (subject.includes('Test Sending Message With Attachment Which Contains Emoji in Filename')) {
       this.strategy = new SaveMessageInStorageStrategy();
+    } else if (subject.includes('Test Sending Message With Recipient Name Contains Emoji')) {
+      this.strategy = new SaveMessageInStorageStrategy();
```

Comment:
Probably this one not needed, as now we're checking recipient in shared-tenant endpoint

Reply:
Aha - yes, it is not needed now. I'll remove it.